src/tools/auth0/handlers/default.js

```diff
@@ -40,8 +40,8 @@ export default class DefaultHandler {
 
   getClientFN(fn) {
     if (typeof fn === 'string') {
-      const client = Reflect.get(this.client, this.type);
-      return Reflect.get(client, fn).bind(client);
+      const client = this.client[this.type]
+      return client[fn].bind(client);
```

Comment:
Similarly with [the core code fix](https://github.com/auth0/auth0-deploy-cli/pull/412/files#diff-e3ee2712757491d17ff1b85f517c4260ad50644cc9e76d31263652dec03190f0R52), I opt to swatch the syntax here from `Reflect.get` to the more straightforward object property bracket syntax.